src/types/SyncTrieHashesReq.ts

```diff
@@ -1,7 +1,9 @@
 import { VectorBufferStream } from '../utils/serialization/VectorBufferStream'
 import { TypeIdentifierEnum } from './enum/TypeIdentifierEnum'
+import { verifyPayload } from './ajv/Helpers'
+import { AJVSchemaEnum } from './enum/AJVSchemaEnum'
 
-const cSyncTrieHashesReqVersion = 1
+export const cSyncTrieHashesReqVersion = 1
```

Comment:
This variable does not need to be exported. I don't see it being used anywhere outside the file. Please change it.

Reply:
This is resolve